modules/misc/xdg-autostart.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+{
+  meta.maintainers = with maintainers; [ jD91mZM2 ];
+
+  options.xdg.autoStart = mkEnableOption ''
+    Automatically start all desktop items in XDG autostart directories (using dex)
```

Comment:
This doesn't get rendered as you think it does: `"Whether to enable <text>."`

Reply:
Oh, thanks